src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 1 failures in `PowerShell-CI-macos`
Experimental Feature: && and || operators - Feature-Enabled.Does not flatten nested values in assignment
```powershell
The term 'Write-Object' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1: line 170
170:         $x = $y = $z = Write-Object 1 && Write-Object @(@(2,3))

```